generator/model.go

```diff
@@ -1363,7 +1363,7 @@ func (sg *schemaGenContext) makeGenSchema() error {
 		return nil
 	}
 	if Debug {
-		log.Println("after shortcuit named ref")
```

Comment:
this is a misspelling of short circuit, not shortcut

Reply:
OK